Library/Formula/apache-forrest.rb

```diff
@@ -4,6 +4,8 @@ class ApacheForrest < Formula
   url "https://www.apache.org/dyn/closer.cgi?path=forrest/apache-forrest-0.9-sources.tar.gz"
   sha256 "c6ac758db2eb0d4d91bd1733bbbc2dec4fdb33603895c464bcb47a34490fb64d"
 
+  depends_on :java
```

Comment:
All right. I think (imho) this is good to go as is if the brew-test-bot tests pass.